src/inc/operators.hpp

```diff
@@ -37,23 +37,3 @@ constexpr bool operator!=(const SMALL_RECT& a, const SMALL_RECT& b) noexcept
 {
     return !(a == b);
 }
-
-constexpr bool operator==(const std::wstring& wstr, const std::wstring_view& wstrView)
-{
-    return (wstrView == std::wstring_view{ wstr.c_str(), wstr.size() });
-}

```

Comment:
why's it still safe to get rid of these? are they just builtin now as of C++17 or so?

Reply:
I'm fairly certain about that. Otherwise it wouldn’t compile…

Reply:
Yep, a comparison of `std::wstring_view` with `std::wstring_view` compiles to the same assembly as `std::wstring` and `std::wstring_view`.